src/universal/styles/theme/brand.js

```diff
@@ -6,7 +6,17 @@ const brand = {
   greyBlue: '#676c8a', // Purple
   darkishPink: '#c62575', // Pink
   eggShell: '#fffcc6', // Cream
-  darkCyan: '#098d8f' // Teal
+  darkCyan: '#098d8f', // Teal
+
+  // From https://github.com/ParabolInc/marketing/blob/aa958f36a9320957e789309dc92e8fa7345c7a01/assets/src/styles/variables/_colors.scss
+  new: {
```

Comment:
I’m ok with this going in, but when I take on #1766 I will clean this up, removing the old values, and not using the “new” object, but rather those will be the new values moving forward.

Reply:
Hey that sounds perfect! My approach here was to deliver this page without disrupting existing styles. Looking forward to #1766!